shell/plugins/steve/subscribe.js

```diff
@@ -32,10 +32,9 @@ import { keyForSubscribe } from '@shell/plugins/steve/resourceWatcher';
 import { waitFor } from '@shell/utils/async';
 import { WORKER_MODES } from './worker';
 import pAndNFiltering from '@shell/utils/projectAndNamespaceFiltering.utils';
-
-import { BLANK_CLUSTER } from '@shell/store/index.js';
 import { STORE } from '@shell/store/store-types';
 
+const BLANK_CLUSTER = '_';
```

Comment:
The `BLANK_CLUSTER` definition should be moved from `shell/store/index.js` to a definitions file rather than a new definition here. 

Think `shell/store/store-types.js` would be the best place